src/main/java/com/client/core/base/tools/propertyeditors/CustomDateTimeEditor.java

```diff
@@ -8,21 +8,29 @@
 
 import java.beans.PropertyEditorSupport;
 
+/**
+ * Custom property editor for joda DateTime. Used for converting DateTime values for spring MVC
+ *
+ * @author magnus.palm
+ *
+ */
 public class CustomDateTimeEditor extends PropertyEditorSupport {
 
     private final DateTimeFormatter customDateTimeFormatter;
 
     private final DateTimeFormatter iso8601DateTimeFormatter;
 
+    private final static String iso8601Format = "yyyy-mm-dd'T'HH:MM:SS.SSS";
```

Comment:
What's the purpose of this field?

Reply:
I have absolutely no idea how this got here